<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BirimListeFormu</class>
 <widget class="QWidget" name="BirimListeFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="grpArama">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Alan Secimi</string>
   </property>
   <widget class="QRadioButton" name="rbBirimKodu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Birim Kodu</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbBirimAdi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Birim Adi</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbTumu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tümü</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbBirimAdresi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Birim Adresi</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tblBirimListesi">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>731</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpArama_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Arama Turu</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arama Metni</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leArama">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAra">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ARA</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpArama_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Arama Turu</string>
   </property>
   <widget class="QRadioButton" name="rbIleBaslayan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ile Baslayan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbIleBiten">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ile Biten</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbIceren">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Iceren</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leArama</sender>
   <signal>textChanged(QString)</signal>
   <receiver>btnAra</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
